### DISCUSSION TITLE
More layers for events in the Chart editor

### DISCUSSION THREAD

**Discussion by LocalVarBright** [FunkinCrew/Funkin#3941] [Ideas]
In the current chart editor, there's only one layer/section for placing events, and with events like zoom, focus camera occurring on the same beat, its really messy to look through them. So I'm suggesting to add upto 3 - 5 layers to support multiple events in one beat. I find it finnicky to either place the events just a very small time distance away from each other, or just stack the events in one square. (plus there's this bug where a ghost event appears which doesn't get deleted and does freaky stuff).

So yeah. More layers.

![image](https://github.com/user-attachments/assets/c0f0a1d7-26c3-4a76-a6fe-d464195069e0)

**Comment by MrMadera** [comment]
that sounds pretty cool! it could also be implemented like psych engine does, allowing you to add as many events as you want and then switch between them
![image](https://github.com/user-attachments/assets/e7b34eb6-0c38-489c-ba3d-0b1e9e34c17c)

**Reply by THEkatinamicrowave** [reply]
ngl I think that every engine except for V-Slice does this. Here's my Live Codename Engine Reaction:
![image](https://github.com/user-attachments/assets/32c94549-d80c-4b7b-9f11-72dc50eb9aa0)
![image](https://github.com/user-attachments/assets/a3a0bdcb-68b5-4793-a241-09fa72cb4382)
